paper.md

```diff
@@ -0,0 +1,65 @@
+
+- `scipy.differentiate`, a sub-package for numerical differentiation of scalar or vector-valued functions of one or more variables,
+- `scipy.integrate.tanhsinh` for quadrature of scalar or vector-valued integrands of one variable,
+- `scipy.integrate.nsum` for summation of real-valued finite or infinite series,
+- `scipy.optimize.elementwise` for finding roots and minimization of a single-input, single output function.
+
+Although the details of the algorithms are inherently distinct, these features rely on a common framework for elementwise iterative methods and share similar interfaces. For example, `scipy.differentiate.derivative` accepts the scalar function to be differentiated as a Python callable, the points at which to evaluate the derivative as an N-dimensional array, and any other parameters of the callable as N-dimensional arrays. `derivative` passes abscissae and parameters to the callable as N-dimensional arrays, adaptively approximating the derivative(s) to achieve specified tolerances. Furthermore, arrays need not be NumPy arrays: the same implementation works with several Python Array API Standard [@arrayapi] compatible arrays, such as PyTorch [@pytorch] tensors and CuPy [@cupy] arrays. These features will dramatically improve performance of end-user applications, and together, they form the backbone of SciPy's new random variable infrastructure.
+
+# Statement of need
+
+Before the release of SciPy 1.15.0, the need for these capabilities was partially met in the scientific Python ecosystem. As popular examples, Numdifftools [@numdifftools] and PyNumDiff [@PyNumDiff] provide tools for numerical differentiation, `quadpy` [@quadpy] and `scipy.integrate` offer several functions for numerical integration, `mpmath`'s [@mpmath] `nsum` function is for series summation, and `scipy.optimize` offers functions for scalar minimization and root finding. However, to the authors' knowledge, the new implementations are unique in that they offer the following advantages.
+
+- Backend-independence: all of these features are "Array API compatible" in the sense that they rely almost entirely on the Python standard libary and calls to Python Array API Standard functions/methods. Consequently, the functions accept objects other than NumPy arrays, such as PyTorch tensors and CuPy arrays, and the calculations are performed by the underlying array library.
+- Speed: the features take full advantage of vectorized user callables, avoiding slow Python loops and the excessive overhead of repeatedly calling compiled code. 
+- Prevalence: SciPy is one of the most popular scientific Python packages. If a scientific Python user needs these features, chances are that they already have SciPy installed, eliminating the need to find and learn a new package.
+- Ease-of-use: the function API reference is thorough, and the interfaces share common features and operate smoothly with other SciPy functions.
```

Comment:
```suggestion
- Ease-of-use: the API reference for these new functions is thorough, and the interfaces share common features and operate smoothly with other SciPy functions.
```

Perhaps this is on me, but it it took me three seconds to realize I should mentally parse this as "the function (API reference)" instead of "the (function API) reference" and in the context of "Array API" I was wondering what the "function API" is.